base_layer/wallet/src/transaction_service/config.rs

```diff
@@ -24,6 +24,7 @@ use log::*;
 use std::{fmt, time::Duration};
 
 const LOG_TARGET: &str = "wallet::transaction_service::config";
+pub const NUM_CONFIRMATIONS_REQUIRED_DEFAULT: u64 = 3;
```

Comment:
Style Nit but is putting this in a const really neater than all of the info being in the Default trait implementation?

Reply:
I never thought of that, good idea. Thanks @philipr-za